### ISSUE TITLE
docs give 404 for some pages

### ISSUE THREAD

**Issue by Ertan222** [nextauthjs/next-auth#7282]
### What is the improvement or update you wish to see?

In docs, some pages doesn't exit and gives 404. I know, next recently change to authjs. Maybe u forget to changes route names or other things. Maybe docs are refactoring for new auth.js but normally says "we are migrating from old docs, if you are looking for this go to old page" but like ı said it gives me 404 and no message. 

![Screenshot from 2023-04-17 10-57-45](https://user-images.githubusercontent.com/62712555/232421872-8e2561c7-c0d6-4061-a150-7c65aee79c31.png)

I am so sorry if this already known but couldn't find an issue or pr about it. 

### Is there any context that might help us understand?

I am using Brave browser, maybe it's browser related but tried with librewolf as well. I didn't block JS obviously.

**Comment by Ertan222** [comment]
> What's the URL? And how did you end up on it?
@balazsorban44 

![Screenshot from 2023-04-17 13-00-20](https://user-images.githubusercontent.com/62712555/232452323-9103c327-1925-4254-aa9a-b295cef46d59.png)

While tried to search from search bar.

**Comment by balazsorban44** [comment]
Hmm, so it's an Algolia indexing issue then, not an actual broken link in this case. We will have a look.

Here is the correct page https://authjs.dev/guides/basics/role-based-access-control#persisting-the-role

**Comment by Ertan222** [comment]
Thanks, ı was actually looking for another page but some other pages gave this error. I just send an example URL. Like this Zoom one: https://authjs.dev/reference/oauth-providers/zoom#example

You don't have to give me real zoom link btw, ı can find from source code if ı need until someone fix it.

**Comment by balazsorban44** [comment]
Yeah, it definitely sounds like Algolia is out-of-sync here/stopped indexing. <picture data-single-emoji=":sad:" title=":sad:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/sad/428220ce479729c2.gif" alt=":sad:" align="absmiddle"></picture>
